setup-dynamodb-tables.sh

```
@@ -0,0 +1,98 @@
+echo ""
+
+# Table 1: featureFlag
+echo "Creating table: featureFlag"
+aws dynamodb create-table \
+  --table-name featureFlag \
+  --attribute-definitions \
+    AttributeName=id,AttributeType=S \
+  --key-schema \
+    AttributeName=id,KeyType=HASH \
+  --billing-mode PAY_PER_REQUEST \
+  --region $REGION \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)